contrib/linux/bash-completion/dosbox

```diff
@@ -0,0 +1,183 @@
+# bash completion for dosbox                               -*- shell-script -*-
```

Comment:
I think this completion is missing pretty important feature: autocompletion of directory or DOS executable file.

Example:

    $ dosbox <tab><tab>

Should print all directories, and files with names ending with .exe, .com, and .bat (case-insensitively). Only single file/directory can be passed this way.

As for style: when comparing to completions present on my system, it seems like 90% of files use 4 spaces - so I suggest following the same style.

And remove vim modeline, please - all completion files on my system *end* with:

    # ex: ts=4 sw=4 et filetype=sh

and use manpage name in first line:

    # dosbox(1) completion                                     -*- shell-script -*-

Reply:
*`dosbox <tab><tab>`* **should** list dirs and exe,bat,com already
This does assume that you are in a parent dir. with subdirs and/or a dos exec. file

I could add a path hint, i.e. `../    /`
I guess adding a 'default' game dir path would be better than /

Reply:
Tested and working as expected.
Sitting inside a DOS game directory:

``` text
[08:09:59@server shamus] ls -1
dosbox.conf
file_id.diz
game.dmr
help.com
list.txt
log1.txt
log2.txt
log.txt
shamus.com
start.bat
```

**`dosbox <tab><tab>`** brings up arguments plus all DOS-executables (help.com, shamus.com, and start.bat): `--printconf   -resetconf    -erasemapper  --MANPAGE     --help        -startmapper  -securemode   -forcescaler  -machine      -exit         -conf         start.bat     help.com`

**`dosbox s<tab>`** lets me pick from shamus.com and start.bat. 

Resolving.